docs/log-collector/README.md

```diff
@@ -43,6 +45,34 @@ tvk-log-collector collects the logs, config and events of resources. Pod Logs ca
   kubectl krew uninstall tvk-log-collector
   ```
 
+#### 2. Without `krew`:
+
+1. List of available releases: https://github.com/trilioData/tvk-plugins/releases
+2. Choose a version of log-collector plugin to install and check if release assets have log-collector plugin's package
+   [log-collector_${version}_${OS}_${ARCH}.tar.gz] for your desired OS & Architecture.
+3. Set env variable `version=v1.x.x` [update with your desired version].
```

Comment:
https://github.com/trilioData/tvk-plugins/releases/latest is also supported. Can we make the script fallback to latest release in case version is not given?

Reply:
Yes, updated doc. Initially, I was hesitant as `pre-release` marked release won't be tagged `latest` so `latest` tag could be way older.